Fabric.Terminology.API/Scripts/Terminology-Install-Utilities.psm1

```diff
@@ -182,6 +182,7 @@ function Get-TerminologyConfig {
         [String] $IisUserName,
         [SecureString] $IisUserPassword,
         [String] $AppName,
+        [String] $AppEndpoint,
```

Comment:
I dont see the change to `Install-Terminology.ps1` to pass in this parameter

Reply:
Added.  I was testing with a value in the install.config and missed that.  Thanks.